script/ci

```diff
@@ -1,2 +1,7 @@
 #!/usr/bin/env bash
+
+pushd cdk > /dev/null || exit
+./script/ci
+popd > /dev/null
```

Comment:
A slightly easier version of this which I've seen in other repos is:

```bash
(
  cd cdk
  ./script/ci
)
```

Reply:
ah wow, nice.